superset-frontend/src/dashboard/components/PropertiesModal/index.tsx

```diff
@@ -161,10 +161,15 @@ const PropertiesModal = ({
 
       form.setFieldsValue(dashboardInfo);
       setDashboardInfo(dashboardInfo);
-      setJsonMetadata(metadata ? jsonStringify(metadata) : '');
       setOwners(owners);
       setRoles(roles);
       setColorScheme(metadata.color_scheme);
+
+      // temporary fix to remove positions from dashboards' metadata
+      if (metadata?.positions) {
+        delete metadata.positions;
```

Comment:
This is to ensure that dashboards that are being affected by an excessively large positions object should work properly. This check can be likely removed in the near future.

Reply:
sorry, why would we be able to remove this in the future? or actually, why do we need this at all if we're removing the attribute in the backend below?

Reply:
@etr2460 all the dashboards that have been saved after the changes were introduced have the `positions` object appearing in their metadata inside the properties modal. I could not reproduce the error in the first place, so I wasn't sure whether that might also cause any problem. This just ensures it won't ever appear in the metadata to the user. I think it can be removed in the future assuming most dashboards will be then be updated and their `positions` object removed in the backend when saved.

Reply:
i see, so this is an attempt to fix up the potentially broken json_metadata. makes sense, thanks!